includes/wcpdf-functions.php

```diff
@@ -320,11 +320,24 @@ function wcpdf_deprecated_function( $function, $version, $replacement = null ) {
 
 /**
  * Logger function to capture errors thrown by this plugin, uses the WC Logger when possible (WC3.0+)
+ * 
+ * @param string         $message
+ * @param string         $level
+ * @param \Exception     $e
+ * @param Order_Document $document
+ * 
+ * @return void
  */
-function wcpdf_log_error( $message, $level = 'error', $e = null ) {
+function wcpdf_log_error( string $message, string $level = 'error', \Exception $e = null, WPO\WC\PDF_Invoices\Documents\Order_Document $document = null ): void {
```

Comment:
I think `$e` can be a lot more than only `\Exception`. Maybe we should not pass it here, it can be added to the `$message` where is called.

Reply:
@alexmigf I think that it won't hurt if we leave it here (but remove the `\Exception` parameter type). It find it useful in `catch` blocks where we have an Exception already defined and ready to be logged. What do you think?

Reply:
Ok, let's try that. You can pass it as `object`.